acquisition.py

```diff
@@ -96,3 +108,144 @@ def get_reward_batch(self, inputs_af_base):  # inputs_af = list of ...
                 outputs = self.proxy.model(inputs, None)
 
         return outputs
+
+
+class AcquisitionFunctionUCB(AcquisitionFunctionBase):
+    def __init__(self, config, proxy):
+        super().__init__(config, proxy)
+
+    def load_best_proxy(self):
+        super().load_best_proxy()
+
+    def get_reward_batch(self, inputs_af_base):  # inputs_af = list of ...
+        super().get_reward_batch(inputs_af_base)
+
+        inputs_af = list(map(torch.tensor, inputs_af_base))
+        input_afs_lens = list(map(len, inputs_af_base))
+        inputs = pad_sequence(inputs_af, batch_first=True, padding_value=0.0)
+
+        self.load_best_proxy()
+        self.proxy.model.eval()
```

Comment:
To activate dropout, you'll have to do chose self.proxy.model.train(), no ?

Reply:
Right sorry, my bad, I will correct that